src/makeRobotAccountant.js

```diff
@@ -7,6 +7,17 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  return (a) => {
+    return (b) => {
+      count++;
+      if (count < 4) {
+        return a + b;
+      }
+      return count % 2 !== 0 ? a + b : "Bzzz... Error!";
+    };
+  };
```

Comment:
```suggestion
  let count = 0;

  return (a) => (b) => {
    count++;
    
    if (count < 4) {
      return a + b;
    }
    
    return count % 2 !== 0 ? a + b : "Bzzz... Error!";
  };
```

You can use implicit return here. But your variant is workable too.

Reply:
Thanks!